tests/IssueDetailsTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace tests;
+
+use ITBugTracking\Entities\IssueDetails;
+use PHPUnit\Framework\TestCase;
+use TypeError;
+
+class IssueDetailsTest extends TestCase
+{
+    public function testIssueDetails_JsonSerialize_success()
+    {
+        $issueDetails = new IssueDetails();
+
+        $issueDetails->id = 5;
+        $issueDetails->title = "A Title";
+        $issueDetails->summary = "This is a description.";
```

Comment:
It's not a description because the field you're updating is called 'summary'... attention to details.

Reply:
done